constraint/pkg/client/drivers/to_result.go

```diff
@@ -74,20 +73,10 @@ func ToResult(constraints map[ConstraintKey]*unstructured.Unstructured, r rego.R
 		Kind: keyMap["kind"],
 		Name: keyMap["name"],
 	}
-
 	constraint := constraints[key]
+
 	// DeepCopy the result so we don't leak internal state.
 	result.Constraint = constraint.DeepCopy()
 
-	enforcementAction, found, err := unstructured.NestedString(constraint.Object, "spec", "enforcementAction")
-	if err != nil {
-		return nil, err
-	}
-	if !found {
-		enforcementAction = apiconstraints.EnforcementActionDeny
```

Comment:
where do we test if `enforcementAction` and `scopedEnforcementActions` are both not provided

Reply:
This test is here - [e2e_test.go#L560](https://github.com/open-policy-agent/frameworks/pull/403/files#diff-7ba8c60c42c538a187e71a73932b6a6a7535ad0a5e8c06e3c50e8b02fc772343R560)